### ISSUE TITLE
Read new genesis.json

### ISSUE THREAD

**Issue by vaibhavchellani** [thehubbleproject/hubble-commander#94]
It would be great if we can feed the `genesis.json` spit by the `contracts` to commander.

## Example genesis.json

```JSON
{
    "parameters": {
        "MAX_DEPTH": 20,
        "MAX_DEPOSIT_SUBTREE_DEPTH": 2,
        "STAKE_AMOUNT": "0",
        "BLOCKS_TO_FINALISE": 40320,
        "MIN_GAS_LEFT": 10000,
        "MAX_TXS_PER_COMMIT": 32,
        "USE_BURN_AUCTION": false,
        "DONATION_ADDRESS": "0x00000000000000000000000000000000000000d0",
        "DONATION_NUMERATOR": 7500,
        "GENESIS_STATE_ROOT": "0x44a6d974c75b07423e1d6d33f481916fdd45830aea11b6347e700cd8b9f0767c"
    },
    "addresses": {
        "paramManager": "0x5FbDB2315678afecb367f032d93F642f64180aa3",
        "frontendGeneric": "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512",
        "frontendTransfer": "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0",
        "frontendMassMigration": "0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9",
        "frontendCreate2Transfer": "0xDc64a140Aa3E981100a9becA4E685f962f0cF6C9",
        "nameRegistry": "0x5FC8d32690cc91D4c39d9d3abcBD16989F875707",
        "blsAccountRegistry": "0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6",
        "tokenRegistry": "0x610178dA211FEF7D417bC0e6FeD39F05609AD788",
        "transfer": "0x9A676e781A523b5d0C0e43731313A708CB607508",
        "massMigration": "0xA51c1fc2f0D1a1b8494Ed1FE312d7C3a78Ed91C0",
        "create2Transfer": "0x959922bE3CAee4b8Cd9a407cc3ac1C251C2007B1",
        "chooser": "0x0165878A594ca255338adfa4d48449f69242Eb8F",
        "exampleToken": "0x68B1D87F95878fE05B998F19b66F4baba5De1aed",
        "spokeRegistry": "0x4ed7c70F96B99c776995fB64377f0d4aB3B0e1C1",
        "vault": "0xa85233C63b9Ee964Add6F2cffe00Fd84eb32338f",
        "depositManager": "0x7a2088a1bFc9d81c55368AE168C2C02570cB814F",
        "rollup": "0xc5a5C42992dECbae36851359345FE25997F5C42d",
        "withdrawManager": "0xc3e53F4d16Ae77Db1c982e75a937B9f60FE63690"
    }
}
```

**Comment by vaibhavchellani** [comment]
This would be a great first issue too.

Any interest @ChihChengLiang?

**Comment by ChihChengLiang** [comment]
yeah, happy to take this

**Comment by ChihChengLiang** [comment]
Fixed in #96
